tests/Symfony/Installer/Tests/IntegrationTest.php

```diff
@@ -106,6 +106,12 @@ public function provideSymfonyInstallationData()
                 '/.*Symfony 2\.5\.6 was successfully installed.*/',
                 '/Symfony version 2\.5\.6 - app\/dev\/debug/',
             ),
+
+            array(
+                '2.7-BETA1',
```

Comment:
This version should fail while `2.7.0-BETA1` should succeed (I think it would be good to have tests for both versions).